score-xml/src/test/java/io/cloudslang/content/xml/actions/InsertBeforeTest.java

```diff
@@ -75,10 +75,10 @@ public void testInvalidElement() {
 
         Map<String, String> result = insertBefore.execute(xml, "", xPathQuery, xmlElement, "false");
 
-        Assert.assertEquals(Constants.FAILURE, result.get(Constants.OutputNames.RESULT_TEXT));
+        Assert.assertEquals(Constants.FAILURE, result.get(Constants.Outputs.RESULT_TEXT));
```

Comment:
@tethryus remove the Assert. and make sure that you imported same as in the other test classes